src/include/sof/debug/panic.h

```diff
@@ -31,10 +31,22 @@ void panic_dump(uint32_t p, struct sof_ipc_panic_info *panic_info,
 	SOF_NORETURN;
 void __panic(uint32_t p, char *filename, uint32_t linenum) SOF_NORETURN;
```

Comment:
Is this declaration still needed in the Zephyr case? (__panic)

Reply:
nope, thanks now fixed.